packages/dds/tree/src/shared-tree-core/messageFormat.ts

```diff
@@ -24,11 +24,20 @@ export interface Message {
 	 * The changeset to be applied.
 	 */
 	readonly changeset: JsonCompatibleReadOnly;
+
+	/**
+	 * The version of the message. This controls how the message is encoded.
+	 *
+	 * This was not set historically and was added before making any breaking changes to the format.
+	 * For that reason, absence of a 'version' field is synonymous with version 1.
```

Comment:
Our other codecs (at least some of them) start with version 0, is there a reason not to do that here? Should we try to be consistent?

Reply:
All of the persisted explicit versions are 1 today.

Changing the code for all implicit versions to also begin with 1 seems like a good idea, I could do it here while I'm touching things, or as a follow-up.

Reply:
Happy for it to happen separately.